scheduler/pkg/kafka/pipeline/kafkamanager.go

```diff
@@ -246,9 +246,9 @@ func (km *KafkaManager) Infer(
 		span.End()
 	}()
 	km.mu.RUnlock()
-	logger.Debugf("Waiting for response for key %s", key)
+	logger.Debugf("Waiting for response for request id %s", requestId)
```

Comment:
do we want to log `resourceName` as well?

Reply:
Added in https://github.com/SeldonIO/seldon-core/pull/5275/commits/6dc56b2cea9c532879e0f4c2bf6e77c812999411